pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
@bparees I'm fine with swapping "args" to "cmd", also maybe to allow future expansion of the postCommit hook we can wrap this up in:

```json
{
  "postCommit": {
    "runImage": {
      "cmd": ["rake test"]
    }
  }
}
```

This allow us to add webhooks or something else into postCommit.